File: app/src/main/java/com/bq/zowi/models/Achievement.java
package com.bq.zowi.models;

/* JADX INFO: loaded from: classes.dex */
public class Achievement {
    public String id;
    public String type;
    public boolean unlocked;

    public enum Id {
        crusaito,
        flapping,
        shake_leg,
        tip_toe,
        jitter,
        ascending_turn,
        swing,
        super_happy,
        sleepy,
        fart,
        confused,
        in_love,
        angry,
        anxious,
        magic,
        wave,
        mouths_editor
    }

    public enum Type {
        movement,
        animation,
        game
    }

    public Achievement(String id, String type, boolean unlocked) {
        this.id = id;
        this.type = type;
        this.unlocked = unlocked;
    }
}

File: zowi-core/src/main/kotlin/com/bq/zowi/models/Achievement.kt
package com.bq.zowi.models

data class Achievement(
    val id: String,
    val type: String,
    var unlocked: Boolean = false
) {
    enum class Id {
        crusaito, flapping, shake_leg, tip_toe, jitter, ascending_turn,
        swing, super_happy, sleepy, fart, confused, in_love, angry,
        anxious, magic, wave, mouths_editor
    }

    enum class Type { movement, animation, game }
}
